source/styles.json

```diff
@@ -0,0 +1,66 @@
+{
+  "rows": {
+
+            "numRows":"3",
+            "translation": "[50, 150]",
+            "itemSize": "[ 2000, 350 ]",
+            "showRowLabel": "[ true ]",
+            "rowLabelOffset": "[ [0, 15], [0, 15], [0, 15] ]",
+            "drawFocusFeedback":true,
+            "vertFocusAnimationStyle":"fixedFocusWrap",
+            "rowFocusAnimationStyle": "fixedFocusWrap",
+            "visible": true,
+            "rowItemSize":"[[500, 285]]",
+            "itemSpacing":"[0, 80]",
+            "rowItemSpacing": " [[30, 0]]"
+          },
+
+  "video":{
+
+              "width":"1920",
+              "height":"1080",
+              "visible":false
+          },
+
+  "detail":{
+              "visible":false
+              },
+
+  "itemLayout": {
+                "layoutDirection": "vert",
+                "vertAlignment":"top",
+                "translation":"[0, 0]"
+                },
+
+    "itemLabel":{
+                  "height": "60",
+                  "width": "0",
+                  "horizAlign": "left",
+                  "vertAlign": "center",
+                  "translation": "[318,8]"
+                  },
+
+    "leftLayoutGroup":{
+                  "translation":"[450,300]"
+                      },
+
+    "image": {
```

Comment:
same here again, which image?

Reply:
Changed here too. Its the image of the video.